jenkins/run_policies.py

```diff
@@ -79,12 +79,12 @@ def get_custodian_policies(type: str = None):
-os.system(f"podman run --rm --name cloud-governance -e account='PSAP' -e policy=tag_iam_user -e AWS_ACCESS_KEY_ID={AWS_ACCESS_KEY_ID_DELETE_PSAP} -e AWS_SECRET_ACCESS_KEY={AWS_SECRET_ACCESS_KEY_DELETE_PSAP} -e user_tag_operation=update -e SPREADSHEET_ID={SPREADSHEET_ID} -e GOOGLE_APPLICATION_CREDENTIALS={GOOGLE_APPLICATION_CREDENTIALS} -e log_level=INFO quay.io/ebattat/cloud-governance:latest")
+os.system(f"podman run --rm --name cloud-governance -e account='PERF-DEPT' -e policy=tag_iam_user -e AWS_ACCESS_KEY_ID={AWS_ACCESS_KEY_ID_DELETE_PERF} -e AWS_SECRET_ACCESS_KEY={AWS_SECRET_ACCESS_KEY_DELETE_PERF} -e user_tag_operation=update -e SPREADSHEET_ID={SPREADSHEET_ID} -e GOOGLE_APPLICATION_CREDENTIALS={GOOGLE_APPLICATION_CREDENTIALS} -v {GOOGLE_APPLICATION_CREDENTIALS}:{GOOGLE_APPLICATION_CREDENTIALS} -e log_level=INFO quay.io/ebattat/cloud-governance:latest")
+os.system(f"podman run --rm --name cloud-governance -e account='PERF-SCALE' -e policy=tag_iam_user -e AWS_ACCESS_KEY_ID={AWS_ACCESS_KEY_ID_DELETE_PERF_SCALE} -e AWS_SECRET_ACCESS_KEY={AWS_SECRET_ACCESS_KEY_DELETE_PERF_SCALE} -e user_tag_operation=update -e SPREADSHEET_ID={SPREADSHEET_ID} -e GOOGLE_APPLICATION_CREDENTIALS={GOOGLE_APPLICATION_CREDENTIALS} -v {GOOGLE_APPLICATION_CREDENTIALS}:{GOOGLE_APPLICATION_CREDENTIALS} -e log_level=INFO quay.io/ebattat/cloud-governance:latest")
+os.system(f"podman run --rm --name cloud-governance -e account='PSAP' -e policy=tag_iam_user -e AWS_ACCESS_KEY_ID={AWS_ACCESS_KEY_ID_DELETE_PSAP} -e AWS_SECRET_ACCESS_KEY={AWS_SECRET_ACCESS_KEY_DELETE_PSAP} -e user_tag_operation=update -e SPREADSHEET_ID={SPREADSHEET_ID} -e GOOGLE_APPLICATION_CREDENTIALS={GOOGLE_APPLICATION_CREDENTIALS} -v {GOOGLE_APPLICATION_CREDENTIALS}:{GOOGLE_APPLICATION_CREDENTIALS} -e log_level=INFO quay.io/ebattat/cloud-governance:latest")
 
 # Gitleaks run on github not related to any aws account
 print("run gitleaks")
 region = 'us-east-1'
 policy = 'gitleaks'
-os.system(f"podman run --rm --name cloud-governance -e policy={policy} -e git_access_token={GITHUB_TOKEN} -e git_repo=https://github.com/redhat-performance -e several_repos=yes -e policy_output=s3://{BUCKET_PERF}/{LOGS}/$region -e log_level=INFO quay.io/ebattat/cloud-governance:latest")
+os.system(f"podman run --rm --name cloud-governance -e policy={policy} e AWS_ACCESS_KEY_ID={AWS_ACCESS_KEY_ID_PERF} -e AWS_SECRET_ACCESS_KEY={AWS_SECRET_ACCESS_KEY_PERF} -e git_access_token={GITHUB_TOKEN} -e git_repo=https://github.com/redhat-performance -e several_repos=yes -e policy_output=s3://{BUCKET_PERF}/{LOGS}/$region -e log_level=INFO quay.io/ebattat/cloud-governance:latest")
```

Comment:
Missing -e in the parameter
Are you sure we need AWS for gitleaks ?

Reply:
yes, we are uploading data to the AWS bucket.